### PR TITLE
Add pre-commit-hooks.yaml

### LINKED ISSUE #1043: Add pre-commit-hooks.yaml
Add pre-commit-hooks.yaml to enable usage of this utility in pre-commit checks

### PR DESCRIPTION
**Pull request by martinrakovec**
Resolves #1043

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-   id: pycodestyle
+    name: pycodestyle
+    description: 'pycodestyle (formerly called pep8) - Python style guide checker'
+    entry: pycodestyle
+    language: python
+    types: [python]
+    require_serial: true
```
Comment (sigmavirus24) [inline .pre-commit-hooks.yaml:7]: I don't believe `require_serial: true` makes any sense for pycodestyle.

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-   id: pycodestyle
+    name: pycodestyle
+    description: 'pycodestyle (formerly called pep8) - Python style guide checker'
```
Comment (sigmavirus24) [inline .pre-commit-hooks.yaml:3]: This should just be `pycodestyle - Python style guide checker` or `Python style guide checker`

File: .pre-commit-hooks.yaml (hunk 1 above)
Comment (martinrakovec) [reply .pre-commit-hooks.yaml]: You are right. I removed it

File: .pre-commit-hooks.yaml (hunk 2 above)
Comment (martinrakovec) [reply .pre-commit-hooks.yaml]: Thanks for your comment. I changed it.

**Comment by asottile** [comment]
this was previously discussed in #821 #979 #1017

**Comment by martinrakovec** [comment]
> 

@asottile Does it mean I should use flake8 instead of pycodestyle in my project and close this PR?

**Comment by asottile** [comment]
yep!

**Pull request closed without merging**